src/rust/engine/engine_cffi/src/cffi_externs.rs

```diff
@@ -20,6 +20,6 @@ pub extern "C" fn initnative_engine() {
 }
 
 #[no_mangle]
-pub unsafe extern "C" fn PyInit_native_engine() -> *mut raw::c_void {
-  wrapped_PyInit_native_engine()
+pub extern "C" fn PyInit_native_engine() -> *mut raw::c_void {
```

Comment:
Because clippy requires method docs otherwise, and this is only intended to be called from C, where an unsafe call would be unchecked anyway.